gateway/handle.go

```diff
@@ -663,7 +663,7 @@ func (gw *Handle) internalBatchHandlerFunc() http.HandlerFunc {
 			stat.Report(gw.stats)
 			goto requestError
```

Comment:
All the errors going to `requestError` should be printing error message. Examples: Line 664 and 680